helpers/base.sh

```diff
@@ -138,27 +138,13 @@ check_dependencies() {
 # Returns:
 #     None
 check_mutually_exclusive_params() {
-    for param in "$@"; do
-        # false is considered as non-empty string, so use empty string
-        # explicitly.
-        if [[ ${!param} == false ]]; then
-            # shellcheck disable=SC2086
-            declare $param=""
-        fi
-    done
-
-    for a in "$@"; do
-        for b in "$@"; do
-            if [[ "${a}" == "${b}" ]]; then
-                continue
-            fi
-
-            if [ ! -z "${!a}" ] && [ ! -z "${!b}" ]; then
-                fatal "${a} and ${b} conflict with each other."
-                exit 1
-            fi
-        done
-    done
+    local output=""
+
+    output="$(${PYTHON} "${PIEMAN_UTILS_DIR}"/check_mutually_exclusive_params.py "$@" 2>&1)"
```

Comment:
`PYTHON` is `$(which python3)` by default. What if python3 is not found?

Reply:
Pieman depends on Python 3, so there must be the `python3` executable on the machine, otherwise the mandatory dependencies won't be satisfied and Pieman won't run (see https://github.com/tolstoyevsky/pieman/blob/master/helpers/base.sh#L97).